### DISCUSSION TITLE
How can I change font size/zoom?

### DISCUSSION THREAD

**Discussion by rtabulov** [TT-ReBORN/Georgia-ReBORN#153] [Q&A] [answered]
Is it possible to increase font size, or maybe change general zoom? Text is too small for me to read comfortably right now

**Comment by TT-ReBORN** [comment, marked as answer]
Hi @rtabulov,

first make sure that the display auto-detection has set the right display mode in top menu `Options` > `Display`.
You can set various font sizes via top menu `Options` > `Font size`.

-TT
